src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -9,10 +9,7 @@ import ChatItemTargetText from './child/ChatItemTargetText';
 import dimen from '../../utils/dimen';
 import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
 import {ANONYMOUS, ANONYMOUS_USER} from '../../hooks/core/constant';
-import {
-  BaseChatItemComponentProps,
-  BaseChatItemTypeProps
-} from '../../../types/component/AnonymousChat/BaseChatItem.types';
+import {BaseChatItemComponentProps} from '../../../types/component/AnonymousChat/BaseChatItem.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `BaseChatItemTypeProps` has been removed. If this type is not used anywhere else in the file, then this change is fine. However, if it's used elsewhere in the code, this could cause a reference error.
```diff
- import {BaseChatItemComponentProps, BaseChatItemTypeProps} from '../../../types/component/AnonymousChat/BaseChatItem.types';
+ import {BaseChatItemComponentProps} from '../../../types/component/AnonymousChat/BaseChatItem.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->